openscap_report/cli.py

```diff
@@ -43,10 +43,26 @@
 EXIT_SUCCESS_CODE = 0
 
 
+class CustomHelpFormatter(argparse.RawTextHelpFormatter):
+    def _format_action_invocation(self, action):
+        if not action.option_strings:
+            metavar, = self._metavar_formatter(action, action.dest)(1)
+            return metavar
+
+        parts = []
+        if action.nargs == 0:
+            parts.extend(action.option_strings)
+        else:
+            default = action.dest.upper()
+            args_string = self._format_args(action, default)
+            parts.append(f'{", ".join(action.option_strings)} {args_string}')
```

Comment:
There is too much braces and quotes on this line, which makes it harder to read. I think it would be more readable if the `", ".join(action.option_strings)` part would be extracted to a variable.